CategoryConstructor/gap/Julia.gi

```diff
@@ -15,6 +15,17 @@ InstallOtherMethod( PreCompose,
     
 end );
 
+##
+InstallOtherMethod( PreComposeList,
+        "for a julia object",
+        [ IsJuliaObject ],
+        
+  function( L )
+    
+    return PreComposeList( ConvertJuliaToGAP( L ) );
+    
+end );
```

Comment:
This should already be handled by https://github.com/homalg-project/CAP_project/blob/6558104e22731280030ea8979f0f0af6b52f63e8/CAP/gap/InstallAdds.gi#L151-L154.

Reply:
Ah, I see!